keps/976-plain-pods/kep.yaml

```diff
@@ -0,0 +1,31 @@
+title: Plain Pods
+kep-number: 976
+authors:
+  - "@alculquicondor"
+status: provisional
+creation-date: 2023-07-31
+reviewers:
+  - "@trasc"
+approvers:
+  - "@ahg-g"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.5"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  beta: "v0.5"
+
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+# No associated feature gates. The feature can be disabled through the Configuration API.
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
+  - my_feature_metric
```

Comment:
Maybe, you forgot to set some value here.

Reply:
If there aren't any metrics, we can remove this.

Reply:
I was leaving for after we resolve all the open questions, but added now.